common/artifact-manager/src/main/java/com/vmware/pscoe/iac/artifact/store/vrang/VraNgPropertyGroupStore.java

```diff
@@ -131,142 +182,137 @@ protected List<String> getItemListFromDescriptor() {
 	}
 
 	/**
-	 * Exports all the property groups, case when the user has not defined any
+	 * Exports all the property groups, case when the user has not listed specific
+	 * items for export.
 	 */
 	protected void exportStoreContent() {
-		logger.info( "Fetched propertyGroups: {}", new Gson().toJson( this.existingPropertyGroups ) );
+		logger.info("Fetched propertyGroups: {}", new Gson().toJson(this.existingPropertyGroups));
 
-		this.existingPropertyGroups.forEach( propertyGroup -> storePropertyGroupOnFileSystem( vraNgPackage, propertyGroup ) );
+		this.existingPropertyGroups
+				.forEach(propertyGroup -> storePropertyGroupOnFileSystem(vraNgPackage, propertyGroup));
```

Comment:
I see no benefit from this change; I also notice in several places the indentation was increased when continuing on new line, not sure if this is the expected formatting.

Reply:
This change is not my preference either, but it comes from automatic formatting. I have used settings from the team onboarding sections.